packages/browser-wallet-api-helpers/src/wallet-api-types.ts

```diff
@@ -250,6 +252,23 @@ interface MainWalletApi {
         message: string | SignMessageObject
     ): Promise<AccountTransactionSignature>;
 
+    /**
+     * Sends a message of the CIS3 contract standard, to the Concordium Wallet and awaits the users action. If the user signs the message, this will resolve to the signature.
+     * Note that if the user rejects signing the message, this will throw an error.
+     * @param contractAddress the {@link ContractAddress} of the contract
+     * @param contractName the {@link ContractName} of the contract
+     * @param entrypointName the {@link EntrypointName} of the contract
+     * @param accountAddress the address of the account that should sign the message
+     * @param message message to be signed. Note that the wallet will prepend some bytes to ensure the message cannot be a transaction. The message should be { @link SignMessageObject }.
+     */
+    signCIS3Message(
+        contractAddress: ContractAddress.Type,
+        contractName: ContractName.Type,
+        entrypointName: EntrypointName.Type,
+        accountAddress: AccountAddressSource,
+        message: SignMessageObject
```

Comment:
**Request change:** Add missing parameter for `nonce` and `expiry` all part of the `PermitMessage` for the parameter of the [`permit` function in CIS-3](https://proposals.concordium.software/CIS/cis-3.html#parameter)